lib/shimmer.js

```diff
@@ -494,6 +500,8 @@ function instrumentPostLoad(agent, nodule, moduleName, resolvedName) {
       return
     }
 
+    const resolvedNodule = instrumentation.isEsm || !esmResolver ? nodule : nodule.default
```

Comment:
We should probably add a comment here about how the default gets into the mix here.

Reply:
good call

Reply:
added comment